test/test_autograd.py

```diff
@@ -6152,6 +6152,18 @@ def test_inplace_view_non_contig(self, device):
         x.sum().backward()
         self.assertEqual(root.grad.tolist(), [[1, 2], [1, 1], [1, 1]])
 
+    def test_mv_grad_stride_0(self, device):
+        # Reference: https://github.com/pytorch/pytorch/issues/38315
+        mat = torch.randn(2, 2, device=device)
+        vec = torch.randn(2, device=device).requires_grad_(True)
```

Comment:
Is that really a Tensor of stride 0?
I think you want: `torch.randn(1, device=device).expand(2).requires_grad_(True)`.
The expand will make the stride be 0.

Reply:
Oops. 
Fixed. Thanks.